app/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,6 @@
     <string name="noAccountYet"><u>No account yet?\nCreate one</u></string>
     <string name="loginFailed">Login failed</string>
     <string name="registerFailed">Registriation failed</string>
+    <string name="card_image_description">Wizard card</string>
```

Comment:
Lets stay with camel case form string names

Reply:
The IDE's default setting is snake case, which is a bit annoying, but I changed it to camel case anyway. In the future, we can maybe consider switching back.